### PR TITLE
Convo blog

### PR DESCRIPTION
**Pull request by grust101**
First blog post so please check that I set the title and various element correctly. Thank you!

### CODE REVIEW

File: _posts/2022/2022-04-01-abbot-conversations.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Track conversations with Abbot"
+description: "We watch the red bubbles in Slack so you don’t have to!"
+tags: [abbot, channels, slack, product]
+excerpt_image:http://img.youtube.com/vi/rt37bpKtook/0.jpg
```
Comment (haacked) [inline _posts/2022/2022-04-01-abbot-conversations.md:5]:
```suggestion
excerpt_image: http://img.youtube.com/vi/rt37bpKtook/0.jpg
```

File: _posts/2022/2022-04-01-abbot-conversations.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    url: 
+    title: Manage and monitor Slack channels with Abbot!
+    alt: Abbot Conversations
```
Comment (haacked) [inline _posts/2022/2022-04-01-abbot-conversations.md:8]:
```suggestion
excerpt_image:
    url: http://img.youtube.com/vi/rt37bpKtook/0.jpg
    title: Manage and monitor Slack channels with Abbot!
    alt: Abbot Conversations
```

File: _posts/2022/2022-04-01-abbot-conversations.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+As your conversation partner in Slack, Abbot monitors and manages shared channels so that you know when a customer conversation needs attention. 
+
+By adding Abbot to a channel, you can configure response time expectations, set up who the first responders are, and leverage Abbot’s smart priority queue to remind your team of conversations so you don’t miss important requests. 
+
+Abbot will ✨automagically✨ measure your team's performance and bandwidth across customer channels to help you identify when a certain account needs more attention or assistance. By reviewing Abbot’s detailed conversation insights and summary across all customer channels, you can make adjustments when needed to maintain smooth conversation flow with your customers.
+
+Each customer relationship is unique, so we’ve made Abbot flexible to allow teams to easily create customized conversation flows, so customers can self-serve when they're looking for help but within a format that works best for your team.
```
Comment (haacked) [inline _posts/2022/2022-04-01-abbot-conversations.md:26]: A bit of a run-on sentence.

```suggestion
Each customer relationship is unique, so we've made Abbot flexible so teams can easily create customized conversation flows. Your team can create flows that let customers self-serve when they look for help, but within a format that works best for your team.
```

**Review by haacked — APPROVED** [review]
:shipit: This is great!

I'm pretty nitpicky about corporate writing so please take all my suggestions as gentle suggestions and not demands or criticism. :smile:

File: _posts/2022/2022-04-01-abbot-conversations.md (hunk 3 above)
Comment (grust101) [reply _posts/2022/2022-04-01-abbot-conversations.md:26]: I like the suggestion except the 'look' part. will revise!

File: _posts/2022/2022-04-01-abbot-conversations.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    title: Track conversations with Abbot
+    alt: Abbot Conversations
+author:
+    avatar: https://avatars.githubusercontent.com/u/18128948?s=400&u=c1e2800aa069189445ce05e50376ad53aa06a793&v=4
+    name: grust101
+---
+
+We are excited to share that Abbot now works alongside customer teams in Slack to manage shared channels and ensure conversations flow smoothly.
+
+[![Screen Shot 2022-04-01 at 1 27 32 PM](https://user-images.githubusercontent.com/18128948/161315857-562a8b69-e691-4f12-bc4a-c8d2f0e80c15.png)](http://www.youtube.com/watch?v=rt37bpKtook "Video Title")
```
Comment (haacked) [inline _posts/2022/2022-04-01-abbot-conversations.md:16]:
```suggestion
[![Image of screenshot from a video about abbot](https://user-images.githubusercontent.com/18128948/161315857-562a8b69-e691-4f12-bc4a-c8d2f0e80c15.png)](http://www.youtube.com/watch?v=rt37bpKtook "Meet Abbot")
```

Comment (haacked) [reply _posts/2022/2022-04-01-abbot-conversations.md:16]: For accessibility reasons, we should always have a descriptive ALT tag and Title for images.

File: _posts/2022/2022-04-01-abbot-conversations.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+excerpt_image:http://img.youtube.com/vi/rt37bpKtook/0.jpg
+    url: 
+    title: Manage and monitor Slack channels with Abbot!
+    alt: Abbot Conversations
+author:
+    avatar: https://avatars.githubusercontent.com/u/18128948?s=400&u=c1e2800aa069189445ce05e50376ad53aa06a793&v=4
+    name: grust101
+---
+
+We are excited to share that Abbot now works alongside customer teams in Slack to manage shared channels and ensure conversations flow smoothly.
```
[the remark this answers is not in the document]
Comment (grust101) [reply _posts/2022/2022-04-01-abbot-conversations.md:14]: Maybe we just take out the 'we are excited to share that..."

**Comment by haacked** [comment]
I did find one issue that needs to be addressed which is the alt tag and title element for the image. Other than that, everything else is my attempt at helpful suggestions. :smile:

File: _posts/2022/2022-04-01-abbot-conversations.md (hunk 3 above)
Comment (haacked) [reply _posts/2022/2022-04-01-abbot-conversations.md:26]: "seek help"

File: _posts/2022/2022-04-01-abbot-conversations.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (haacked) [reply _posts/2022/2022-04-01-abbot-conversations.md:14]: One thing I will say about the "We're excited" part is I don't mind if a post ends with something like "We're excited for you to try it out!" I just don't like to start with that.

**Pull request merged**